modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -4718,7 +4726,7 @@ class CV_EXPORTS LineIterator
 inline
 uchar* LineIterator::operator *()
 {
-    return ptr;
+    return !ptr0 ? 0 : ptr;//when no Mat is attached, ptr is just a dummy address and should not be dereferenced
```

Comment:
"Contact" of this call has been changed.
This patch allows "NULL" as a result. At least this would produces static analyzers issues which need to be resolved after this patch.